### PR TITLE
Get ready for the day section

### PR DESCRIPTION
**Pull request by florestankorp**
Get ready for the day section #25

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-                </div>
-                <div class="col-md-4">
-                    <%= link_to "Host TOC",  sponsor_path, class: "btn btn-lg btn-default center-block" %>
-                </div>
-            </div>
-            <footer>
-                <p>&copy; 2017 Koalas, Inc.</p>
-            </footer>
+        <div class="col-md-4">
+          <%= link_to "Host TOC",  sponsor_path, class: "btn btn-lg btn-default center-block" %>
```
Comment (SarahPAms) [inline app/views/pages/home.html.erb]: "Taste of Code"?

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-            <h2>Beginner Coding Workshop</h2>
+  <div class="centerscreen">
+    <div class="container jumbotron mainscreen-filler">
+      <h1>Taste of Code</h1>
+      <h2>Beginner Coding Workshop</h2>
+    </div>
+    <div class="container">
+      <div class="row">
+        <div class="col-md-4">
+          <%= link_to "Join TCO",  attend_path, class: "btn btn-lg btn-default center-block" %>
```
Comment (SarahPAms) [inline app/views/pages/home.html.erb]: It was mentioned this morning that TCO should be Taste of Code.  Does that apply here?

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
-                </div>
-            </div>
-            <footer>
-                <p>&copy; 2017 Koalas, Inc.</p>
-            </footer>
+        <div class="col-md-4">
+          <%= link_to "Host TOC",  sponsor_path, class: "btn btn-lg btn-default center-block" %>
         </div>
+      </div>
+      <footer>
```
Comment (SarahPAms) [inline app/views/pages/home.html.erb]: Issue #29 (footer partial) will resolve this.

File: app/views/pages/attend.html.erb [hunk 4]
```diff
@@ -2,3 +2,4 @@
 
 <!--  Testimonial partial -->
 <%= render '/shared/testimonials' %>
+<%= render '/shared/get_ready_for_the_day_section' %>
```
Comment (arnoFleming) [inline app/views/pages/attend.html.erb]: `render 'shared/that/file'` will do, don't include that first forward slash! `'/shared`

File: app/views/shared/_get_ready_for_the_day_section.html.erb [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<section>
+  <h1 class="text-center">Get Ready for the Day!</h3>
+  <h2 class="text-center">Know Your Tools
+  </h2>
```
Comment (arnoFleming) [inline app/views/shared/_get_ready_for_the_day_section.html.erb]: move it to previous line

File: app/views/shared/_get_ready_for_the_day_section.html.erb [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  <div class="container">
+    <div class="col-lg-4 text-center">
```
Comment (arnoFleming) [inline app/views/shared/_get_ready_for_the_day_section.html.erb]: you know your bootstrap well, dear sir!

File: app/views/pages/attend.html.erb (hunk 4 above)
Comment (arnoFleming) [inline app/views/pages/attend.html.erb]: maybe make a PR to update all `render` calls to not include that first `/` ?

**Pull request merged**